packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePicker.tsx

```diff
@@ -410,27 +408,21 @@ export const useDatePicker_unstable = (props: DatePickerProps, ref: React.Ref<HT
     disabled: !open,
   });
 
-  const { findFirstFocusable } = useFocusFinders();
+  // When the popup is opened, focus should go to the calendar.
+  // In v8 this was done by focusing after the callout was positioned, but in v9 this can be simulated by using an
+  // a useEffect hook
```

Comment:
```suggestion
  // When the popup is opened, focus should go to the calendar.
  // In v8 this was done by focusing after the callout was positioned, but in v9 this can be simulated by using an
  // useEffect hook.
```